testing/trino-tests/src/test/java/io/trino/execution/EventsAwaitingQueries.java

```diff
@@ -78,7 +78,7 @@ MaterializedResultWithEvents runQueryAndWaitForEvents(@Language("SQL") String sq
         }
 
         QueryEvents queryEvents = eventsCollector.getQueryEvents(queryId);
-        queryEvents.waitForQueryCompletion(new Duration(30, SECONDS));
+        queryEvents.waitForQueryCompletion(new Duration(3, SECONDS));
```

Comment:
How is that possible that longer timeout fixed the issue? If there was no event listener registered then it should not work at all? Are you referring here to a different problem?

According https://github.com/trinodb/trino/pull/13321 it should be 30 seconds.

Reply:
https://github.com/trinodb/trino/pull/13321#issuecomment-1193170863 - no effect but considered an improvement.
FYI - https://github.com/trinodb/trino/pull/13370 - This PR is probably superseded @wendigo @kokosing